tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/person_id_validator_test.py

```diff
@@ -78,14 +76,14 @@ def setUp(self):
             INSERT INTO `{{project_id}}.{{dataset_id}}.observation_ext`
                 (observation_id, src_id, survey_version_concept_id)
             VALUES
-                (100, 'PPI/PM', null),
-                (101, 'PPI/PM', null),
+                (100, 'Portal 1', null),
+                (101, 'Portal 2', null),
                 (102, 'site bar', null),
-                (201, 'PPI/PM', null),
-                (202, 'PPI/PM', null),
+                (201, 'Portal 3', null),
+                (202, 'Portal 4', null),
                 (203, 'site baz', null),
                 (301, 'site raz', null),
-                (302, 'PPI/PM', null)
+                (302, 'Portal 5', null)
```

Comment:
Does the test pick up the sites properly? They might need to be renamed to `EHR site ...`

Reply:
Thanks for explaining that the query will pick up only things that are not ~'portal'. The other values will be picked up automatically and don't need to be edited for the test.